src/utests/utf_baselib_jni/java/src/com/jpmc/swblocks/baselib/test/JavaBridge.java

```diff
@@ -0,0 +1,36 @@
+package com.jpmc.swblocks.baselib.test;
+
+import java.nio.ByteBuffer;
+
+public class JavaBridge {
+
+    static public void dispatch(ByteBuffer inputBuffer, ByteBuffer outputBuffer) throws Exception {
```

Comment:
Yes, Exception shouldn't be thrown from 'dispatch'. All checked Java exceptions should be handled on the Java side,